src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -307,11 +323,10 @@ public final boolean load(SectionNode sectionNode) {
 	 *
 	 * @param backupInterval the backup interval.
 	 */
-	public void startBackupTask(Timespan backupInterval) {
+	public void startBackupTask(Timespan backupInterval, boolean purge, int tokeep) {
```

Comment:
Like above, purge isn't a great name. Something like "removeExtra" would be better. Also capitalize toKeep